.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,31 @@
+<!-- Please answer these questions before submitting your issue. Thanks! -->
+
+### What version of Go are you using (`go version`)?
+
+<pre>
+$ go version
+
+</pre>
+
+### What version of Badger are you using?
+
+
+### Does this issue reproduce with the latest release?
```

Comment:
We can also ask to reproduce it on latest master.

Reply:
Done